core/src/test/java/org/jclouds/rest/binders/BindToJsonPayloadTest.java

```diff
@@ -47,6 +48,9 @@ public void testMap() throws SecurityException, NoSuchMethodException {
 
    }
 
+   // TODO: fails with Failed making field 'java.io.File#path' accessible; either increase its visibility or write a custom TypeAdapter for its declaring type.
+   // This is serializing a File which we don't actually care about.  Just pick a different type instead?
+   @Ignore
```

Comment:
Any other suggesions to address this?  I might have misunderstood what gson is/should be doing.